bin/datatool/Cargo.toml

```diff
@@ -10,15 +10,22 @@ path = "src/main.rs"
 [dependencies]
 strata-key-derivation.workspace = true
 strata-primitives.workspace = true
-strata-sp1-guest-builder = { path = "../../provers/sp1" }
+strata-risc0-guest-builder = { path = "../../provers/risc0", optional = true }
+strata-sp1-guest-builder = { path = "../../provers/sp1", optional = true }
 
 anyhow.workspace = true
 argh.workspace = true
 bech32 = "0.11.0"
 bitcoin = { workspace = true, features = ["std"] }
+bytemuck = "1.21.0"
```

Comment:
since bytemuck is only used with risc0, should we make it optional and feature-gated with risc0?